.github/pull_request_template.md

```diff
@@ -10,8 +10,8 @@ You will be automatically asked to sign the contributors license before the PR i
 
 ##### Checklist
 
-- [ ] Issue filed at https://unicode-org.atlassian.net :  ICU-______
-- [ ] Update PR title to include Issue number
+- [ ] Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
```

Comment:
the point is for people to create a new ticket. https://unicode-org.atlassian.net/browse/ICU-_____ is a dead link.

Reply:
I guess you can still create a ticket from there. I'll approve.

Reply:
I suppose we might have a direct link as well?
Maybe add another step?
- [ ] Issue filed. [Create Issue](https://unicode-org.atlassian.net/secure/CreateIssue!default.jspa?selectedProjectId=1)
- [ ] Link to issue: https://unicode-org.atlassian.net/browse/ICU-_____
- [ ] ...etc..

Reply:
I think it's obvious enough where to go to file an issue.  Plus, the "Create Issue" link doesn't work unless you look at the Markdown preview.